src/main/java/cs/ualberta/cmput402/tictactoe/board/Board.java

```diff
@@ -116,7 +131,32 @@ public void printBoard(){
         }
     }
 
-    public Player getCurrentPlayer() {
+    public void scoreBoard(){
+
+        currentPlayer = getWinner();
+
+        if (currentPlayer == Player.X){
+            scoreX++;
+        }
+
+        if(currentPlayer == Player.O){
+            scoreY++;
+        }
+
+    }
+
+    public void printScoreBoard() {
+
+	  	  System.out.println("Score Board");
+	      System.out.println("----------");
+	      System.out.println("X has won: " + scoreX + " times, and lost: " + scoreY + " times");
+	      System.out.println("O has won: " + scoreY + " times, and lost: " + scoreX + " times");
+          System.out.println("Number of ties: " + ties );
+	      System.out.println("----------");
```

Comment:
Indentation issues

Reply:
Fixed these indentation inconsistencies in 9f96f5f